src/components/CopyUrlButton/README.md

```diff
@@ -0,0 +1,4 @@
+## CopyUrlButton
+An implementation of an CopyUrlButton for TPAs
+This Readme was auto-generated and needs to be filled manually
+
```

Comment:
We can remove all README files.
They're not really needed.

Reply:
done